cpp/src/hash/hashing.cu

```diff
@@ -213,7 +221,168 @@ void compute_row_output_locations(size_type * __restrict__ row_partition_numbers
   }
 }
 
-template <bool has_nulls>
+/* --------------------------------------------------------------------------*/
+/** 
+ * @brief Move one column from the input table to the hashed table.
+ * 
+ * @param[in] input_buf Data buffer of the column in the input table
+ * @param[out] output_buf Preallocated data buffer of the column in the output table
```

Comment:
Within `cuDF`, especially the legacy functions as newer APIs favor returning outputs instead of output parameters, the output table/column tends to come right after the input table/column.